### PR TITLE
Fix #4197: Introduce new hints banner in exploration player

### LINKED ISSUE #4197: Update Hints icon
**Is your feature request related to a problem? Please describe.**
We have received feedback from our users that the hint bulb is not clearly discoverable in the app.

**Describe the solution you'd like**
Update the hint icon to display the word "Hint" before the bulb and make it bounce up and down thrice every time a hint is available. Whenever a new hint is available, an additional banner saying "New" should appear on the top right of this button. Please see the lightweight mock below:
image

**Describe alternatives you've considered**
i) Adding help text "Do you need help?" above the hint icon (Trade Off - This can make the screen cluttered, and cause distractions to the user) 
ii) Audio cues, like a pop or tap sound once a new hint is available (Trade Off - This can cause distractions)

**Additional context**
Specs for animation are shared in this doc: https://docs.google.com/document/d/1lrvjulcybrbvMjwIkheI_pnW-k4T7X1FFlEmNZYtVIo

### PR DESCRIPTION
**Pull request by BenHenning**
## Explanation
Fixes #4197

This PR modifies the hints banner in the Exploration Player, adding new bulb and arrow drawables as well as adding logic for each to be shown based on hint state.

The PR ensures that the views have content descriptions for accessibility, and drawables are automirrored for localization in RTL layouts.

Tests have been added to verify that the drawables are shown appropriately.

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

### LTR Demo
https://github.com/user-attachments/assets/e5623ddd-f89d-4eba-a6d0-013b9ec52af4

### RTL Demo
https://github.com/user-attachments/assets/1b3f0679-db82-421e-a8ea-e1f5f669da2f

### Talkback Navigation Demo
https://github.com/user-attachments/assets/cc888d1a-1abe-4bcb-afc8-f130e104029e

### CODE REVIEW

**Comment by BenHenning** [comment]
Still active.

**Comment by BenHenning** [comment]
@anandwana001 this actually should be ready for review, I just haven't finished the PR description (which will probably take more than an hour due to the large number of screenshots & the Espresso test that need to be collected).

Apologies for the lack of PR description, but PTAL at the issue if you need context. If you can get enough context, PTAL at the whole PR as a merge-ready review. I'll make sure the PR description is completed per the template before merging.

**Review by anandwana001 — APPROVED** [review]
Banner LGTM.

**Comment by BenHenning** [comment]
Still active.

**Comment by BenHenning** [comment]
This is being merged without the PR description being completed (#4345 is tracking actually finishing it in at a future time) due to a tight time pressure on finishing this release before I take time off.

Also, @rt4914's review is being deferred per a prior discussion with him (#4266 is tracking the post-merge review).

File: app/src/main/res/drawable/ic_hint_bulb_white_48dp.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<vector android:height="48dp" android:tint="#FFFFFF"
+    android:viewportHeight="24.0" android:viewportWidth="24.0"
+    android:width="48dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <path android:fillColor="#FF000000" android:pathData="M9,21c0,0.55 0.45,1 1,1h4c0.55,0 1,-0.45 1,-1v-1L9,20v1zM12,2C8.14,2 5,5.14 5,9c0,2.38 1.19,4.47 3,5.74L8,17c0,0.55 0.45,1 1,1h6c0.55,0 1,-0.45 1,-1v-2.26c1.81,-1.27 3,-3.36 3,-5.74 0,-3.86 -3.14,-7 -7,-7zM14.85,13.1l-0.85,0.6L14,16h-4v-2.3l-0.85,-0.6C7.8,12.16 7,10.63 7,9c0,-2.76 2.24,-5 5,-5s5,2.24 5,5c0,1.63 -0.8,3.16 -2.15,4.1z"/>
+</vector>
```
Comment (rt4914) [inline app/src/main/res/drawable/ic_hint_bulb_white_48dp.xml:5]: Optional: This file can be re-organised.

**Review by rt4914 — COMMENTED** [review]
Filed on issue https://github.com/oppia/oppia-android/issues/4397 (not part of this PR). Approved.

File: app/src/main/res/drawable/ic_hint_bulb_white_48dp.xml (hunk 1 above)
Comment (BenHenning) [reply app/src/main/res/drawable/ic_hint_bulb_white_48dp.xml:5]: This is being addressed in #4506.

**Review by adhiamboperes — COMMENTED** [review]
PR LGTM! The test coverage is thorough, and I have updated the description per #5038

**Pull request merged**